package.json

```diff
@@ -16,7 +16,7 @@
   "author": "Traceur Authors",
   "license": "Apache License 2.0",
   "engines": {
-    "node": ">=0.10"
+    "node": ">=0.12"
```

Comment:
I think I got confused by a request I got asking me to remove the engine constraint from a package.json so they could use iojs with my library. But if ```"node": ">=0.12"``` works with iojs then im happy.